src/pages/home/ReportScreen.js

```diff
@@ -181,6 +181,7 @@ class ReportScreen extends React.Component {
                                 reportID={reportID}
                                 reportActions={this.props.reportActions}
                                 report={this.props.report}
+                                isFocused
```

Comment:
Why do you need this?

Reply:
same reason  "need to autoFocus on devices that are supported as discussed in the issue."

Reply:
If you go through the history of this Component, you will find that we deliberately block focus on mobile devices. If you could please share a link to the discussion, I can reflect that into the comments.

Reply:
https://github.com/Expensify/App/issues/4678#issuecomment-904413983

Reply:
> If you go through the history of this Component, you will find that we deliberately block focus on mobile devices.
if you want to know more about why we blocked the opened keyboard.